setup_trinity.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+
+setup(
+    name='trinity',
+    # *IMPORTANT*: Don't manually change the version here. Use the 'bumpversion' utility.
+    # NOT CURRENTLY APPLICABLE. VERSION BUMPS MANUAL FOR NOW
+    version='0.1.0-alpha.2',
+    description='The Trinity Ethereum Client',
+    author='Piper Merriam',
```

Comment:
Not trying to belittle your role but I think this should be something like "The Trinity developers" or maybe "Ethereum Foundation" but not sure about any legal implications. There are several other places that are hardcoded to you and @carver that I think should be resolved in a similar way.